docs/src/operate/installation/026_how_to_install_thin_edge_manually.md

```diff
@@ -20,9 +20,7 @@ You will also need to have mosquitto installed. Check your package manager for a
 
 To build from source, download the **Source code (zip)** from the [latest releases page](https://github.com/thin-edge/thin-edge.io/releases/latest).
 
-<p align="center">
-  <img src="./images/manual_installation-download_source_code.png" alt="Sublime's custom image"/>
-</p>
+![Sublime's custom image](../../howto-guides/images/manual_installation-download_source_code.png)
```

Comment:
Sublime?

Reply:
I guess the name of the tool used by the primary author.